tcf_website/static/reviews/new_review.js

```diff
@@ -28,15 +29,16 @@ jQuery(function($) {
             }).appendTo("#subject");
         });
         return this;
-    });
-
-    // Enable subject selector, disable the following
-    $("#subject").prop("disabled", false);
-    $("#course").prop("disabled", true);
-    $("#instructor").prop("disabled", true);
-    $("#semester").prop("disabled", true);
+    })
+        .done(function() { // Second callback
```

Comment:
Not exactly sure what "second" is referring to; also, none of the other callback `done` functions are labelled like this?

Reply:
the `.done()` callback is called after the function callback